app/javascript/flavours/polyam/features/compose/components/emoji_picker_dropdown.jsx

```diff
@@ -10,12 +10,14 @@ import ImmutablePropTypes from 'react-immutable-proptypes';
 import { supportsPassiveEvents } from 'detect-passive-events';
 import Overlay from 'react-overlays/Overlay';
 
+import data from 'flavours/polyam/features/emoji/emoji_data.json';
```

Comment:
Instead of importing data here, the Emoji and EmojiPicker component could be wrapped to use this data.
emoji-mart uses NimbleEmoji and NimblePicker internally anyway.